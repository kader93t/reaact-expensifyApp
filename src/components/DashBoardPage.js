import React from "react"
import ExpensesList from './ExpnesesList'
import ExpenseFilter from "./ExpenseFilter"

const ExpenseDashBoardPage = () => (
    <div>
        Welcom to react Router
        <ExpenseFilter />
        <ExpensesList />
    </div>
);


export default ExpenseDashBoardPage;