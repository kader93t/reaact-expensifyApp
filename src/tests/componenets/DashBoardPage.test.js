import React from 'react'
import DashBoardPage from "../../components/DashBoardPage"
import { shallow } from "enzyme"

test('should render the DashBoardPage ', () => {
    const wrapper = shallow(<DashBoardPage />);
    expect(wrapper).toMatchSnapshot();
})
