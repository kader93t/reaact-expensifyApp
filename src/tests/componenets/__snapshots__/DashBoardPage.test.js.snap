// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the DashBoardPage  1`] = `
<div>
  Welcom to react Router
  <Connect(ExpensesFilters) />
  <Connect(ExpensesList) />
</div>
`;
